<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EnvironmentOptionsWidget</class>
 <widget class="QWidget" name="EnvironmentOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>179</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="3" column="1">
    <widget class="RealControl" name="sunThetaRealControl">
     <property name="suffix">
      <string> deg</string>
     </property>
     <property name="maximum">
      <double>120.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Sun angle from vertical</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Sun azimuthal angle</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="RealControl" name="sunPhiRealControl">
     <property name="suffix">
      <string> deg</string>
     </property>
     <property name="maximum">
      <double>720.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>5.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="useLocalSunDirCheckBox">
     <property name="text">
      <string>Use local sun direction</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RealControl</class>
   <extends>QDoubleSpinBox</extends>
   <header>../qt/RealControl.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
